values/vault.yaml.gotmpl

```diff
@@ -0,0 +1,265 @@
+        # This is true by default
+        preFlightChecks: true
+      kubernetes:
+        secretNamespace: {{ .Environment.Values.vaultNamespace }}
+
+    # A YAML representation of a final vault config file.
+    # See https://www.vaultproject.io/docs/configuration/ for more information.
+    config:
+      storage:
+        file:
+          path: "${ .Env.VAULT_STORAGE_FILE }" # An example how Vault config environment interpolation can be used
+      listener:
+        tcp:
+          address: "0.0.0.0:8200"
+          # Uncommenting the following line and deleting tls_cert_file and tls_key_file disables TLS
+          tls_disable: true
+          #tls_cert_file: /vault/tls/server.crt
+          #tls_key_file: /vault/tls/server.key
+      plugin_directory: /vault/plugins
+      api_addr: http://localhost:8200
+      {{ if .Environment.Values.metrics.enabled }}
+      telemetry:
+        statsd_address: localhost:9125
+      {{ end }}
+
+    serviceMonitorEnabled: {{ .Environment.Values.metrics.enabled }}
+
+    # See: https://github.com/banzaicloud/bank-vaults#example-external-vault-configuration for more details.
+    externalConfig:
+      policies:
+        {{ if .Environment.Values.metrics.enabled }}
+        - name: prometheus
+          rules: path "sys/metrics" {
+            capabilities = ["list", "read"]
+            }
+        {{ end }}
+        - name: allow_secrets
+          rules: path "orchestrate/*" {
+            capabilities = ["create", "read", "update", "delete", "list"]
+            }
+        - name: api_key_manager_demo
+          rules: path "orchestrate/data/{{ .Environment.Values.orchestrateNamespace }}/keys/*" {
+            capabilities = ["create", "read", "update", "delete", "list"]
+            }
+      auth:
+        - type: kubernetes
+          roles:
+            - name: api-key-manager
+              bound_service_account_names: ["api-key-manager", "vault-secrets-webhook", "vault"]
+              bound_service_account_namespaces: ["{{ .Environment.Values.vaultNamespace }}", "{{ .Environment.Values.vaultOperatorNamespace }}", "{{ .Environment.Values.orchestrateNamespace }}"]
+              policies: ["allow_secrets", "api_key_manager_demo"]
+        {{ if .Environment.Values.metrics.enabled }}
+        - type: kubernetes
+          roles:
+            - name: prometheus
+              bound_service_account_names: prometheus
+              bound_service_account_namespaces: {{ .Environment.Values.observabilityNamespace }}
+              policies: prometheus
+        {{ end }}
+
+      secrets:
+        - path: orchestrate
+          type: plugin
+          plugin_name: orchestrate
+          description: Orchestrate key storage gateway
+          config:
+            force_no_cache: true
+            passthrough_request_headers: ["X-Vault-Namespace"]
+
+      plugins:
+        - plugin_name: orchestrate
+          command: orchestrate
+          sha256: {{ .Environment.Values.vaultPlugin.sha256 }}
```

Comment:
it should be fetch https://github.com/ConsenSys/orchestrate-hashicorp-vault-plugin/releases/tag/v0.0.5